webpack/assets/javascripts/react_app/components/statistics/ChartBox.js

```diff
@@ -37,13 +49,15 @@ class ChartBox extends React.Component {
     const Chart = components[type];
     const dataFiltered = chart.data && chart.data.filter(arr => arr[1] !== 0);
     const hasChartData = dataFiltered && dataFiltered.length > 0;
-    const tooltip = hasChartData ? {
-      onClick: this.onClick,
-      title: this.props.tip,
-      'data-toggle': 'tooltip',
-      'data-placement': 'top',
-      className: 'pointer',
-    } : {};
+    const tooltip = hasChartData
+      ? {
+        onClick: this.onClick,
+        title: this.props.tip,
+        'data-toggle': 'tooltip',
```

Comment:
maybe we should use the tooltip react component instead of the jquery one as well?

Reply:
after few attempts -  react-bootstrap `Panel` header doesn't work well with `<OverlayTrigger>`. plus it has a breaking changes new API in the later version. I think it better to handle this in a different PR, because it doesn't related to statistics page performance.

Reply:
can you please add it as an issue that we won't forget about it?

Reply:
@amirfefer please don't forget to add this issue.